addons/events/fnc_playerEvent.sqf

```diff
@@ -21,30 +21,17 @@ Author:
 ---------------------------------------------------------------------------- */
 SCRIPT(playerEvent);
 
-private _unit = missionNamespace getVariable ["bis_fnc_moduleRemoteControl_unit", player];
+private _unit = focusOn;
```

Comment:
We still want the original line
When controlling uavs this will return the UAV AI

- control a uav
- typeOf ace_player = "B_UAV_AI"
- self interact and you can now check the pulse of the AI which makes no sense

Reply:
```suggestion
private _unit = missionNamespace getVariable ["bis_fnc_moduleRemoteControl_unit", player];
```